src/Microsoft.VisualStudio.Editors/package/VBPackage.vb

```diff
@@ -20,7 +24,8 @@ Namespace Microsoft.VisualStudio.Editors
     '*
     '* In the future, we should consider moving to a RegPkg.exe model
     <
-        System.Runtime.InteropServices.Guid("67909B06-91E9-4F3E-AB50-495046BE9A9A"),
+        Guid("67909B06-91E9-4F3E-AB50-495046BE9A9A"),
+        ProvideOptionPage(GetType(GeneralOptionPage), "Projects and Solutions", ".NET Core", 1400, 1400, True),
```

Comment:
Sadly, VB overloads := for named arguments AND attribute property assignment. When you try to use :=, you get errors because the properties are read-only.

---
In reply to: [117557825](https://github.com/dotnet/project-system/pull/2243#discussion_r117557825) [](ancestors = 117557825)